auto-instrument-handler/index.test.js

```diff
@@ -3,47 +3,38 @@ const index = require('./index');
 describe('tracer', () => {
   test('handler call the original function', () => {
     process.env[index.LUMIGO_SWITCH_OFF] = 'TRUE';
+    process.env.LAMBDA_TASK_ROOT = '/var/task';
 
     process.env[index.ORIGINAL_HANDLER_KEY] = '';
     expect(() => index.handler({}, {})).toThrowError(
-      'Could not load the original handler. Are you sure that the handler is correct?'
+      'Bad handler'
```

Comment:
why change the original message?

Reply:
On the contrary - we used to change the original message.
Not we're raising the exact same message as the vanilla runtime :)